TIDALDL-PY/tidal_dl/lang/english.py

```diff
@@ -24,7 +24,7 @@ class LangEnglish(object):
     SETTING_CHECK_EXIST = "Check exist"
     SETTING_ARTIST_BEFORE_TITLE = "ArtistName before track-title"
     SETTING_ALBUMID_BEFORE_FOLDER = "Id before album-folder"
-    SETTING_INCLUDE_EP = "Include single&ep"
+    SETTING_INCLUDE_EP = "Include single&EP"

```

Comment:
```suggestion
    SETTING_INCLUDE_EP = "Include Singles & EPs"
```

Maybe is better in this way. What do you think?

Reply:
Yes, that's a good suggestion! I think it'd be better to refer to the 73rd line and translate it as "Include singles & EPs".